core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -40,24 +40,36 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
               <span ng-if="getAverageRating()">
                 <[getAverageRating() | number:1]>
               </span>
-              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A">
+              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A" class="inactive-summary-tile-attribute ">
               </span>
             </span>
           </li>
 
           <li>
             <span class="fa fa-eye fa-lg" tooltip="<['I18N_LIBRARY_VIEWS_TOOLTIP' | translate]>" tooltip-placement="top">
             </span>
-            <[getNumViews() | summarizeNonnegativeNumber]>
+            <span ng-if="getNumViews()">
+              <[getNumViews() | summarizeNonnegativeNumber]>
+            </span>
+            <span ng-if="!getNumViews()" translate="I18N_LIBRARY_N/A" class="inactive-summary-tile-attribute ">
+            </span>
           </li>
 
           <li>
-            <span>
+            <span ng-if="getLastUpdatedDatetime()">
               <[getLastUpdatedDatetime()]>
             </span>
+            <span ng-if="!getLastUpdatedDatetime()" translate="I18N_LIBRARY_N/A" class="inactive-summary-tile-attribute">
+            </span>
           </li>
         </ul>
       </div>
     </a>
   </md-card>
 </script>
+
+<style>
+.oppia-activity-summary-tile .inactive-summary-tile-attribute {
```

Comment:
Indent this and hte following line by 2.

Also check whether .oppia-activity-summary-tile is used elsewhere in the codebase. If so, it may be better for the first part of the selector to be just, e.g., `exploration-summary-tile`.